### DISCUSSION TITLE
Problem to re play a streaming audio

### DISCUSSION THREAD

**Discussion by CaleoSystems** [bbc/peaks.js#514] [General]
Hello,
I am using Peaks to play audios stored on a remote server served by a NodeJS based streaming server. Everything works fine before the audio ends, when the audio ends and I play it again I can't go forward or back since it always sends it to the beginning. This behavior only occurs with streaming and not with local audio.

Thanks for your help.

**Comment by chrisn** [comment]
Closing, to keep discussion in one place. See #515.
